### ISSUE TITLE
blacklists list is not respected anymore

### ISSUE THREAD

**Issue by borring** [1995eaton/chromium-vim#352]
I'm not sure when it stopped working. It used to work, but now it doesn't

Chrome: 44.0.2403.157 (64-bit linux)
cvim: 1.2.79

**Comment by 1995eaton** [comment]
Could you share your cvimrc?

**Comment by borring** [comment]
Sure thing. Here it is:

	" Settings
	set noautofocus

	set typelinkhints
	let searchlimit = 60
	let scrollstep = 120
	let barposition = "bottom"
	set scalehints
	set incsearch
	set ignorecase
	set smartcase
	set linkanimations
	set nativelinkorder
	let homedirectory="/home/moymoy"

	set changelog

	let highlight="#cccccc"
	let vimcommand="nvim-qt"

	let completionengines = ["google", "amazon", "imdb", "dogpile"]

	let searchalias g = "google"



	" Open all of these in a tab with `gnb` or open one of these with <N>goa where <N>
	let qmark a = ["http://www.reddit.com/r/android", "http://www.reddit.com/r/linux", "http://www.reddit.com/r/vim", "http://www.reddit.com/r/neovim"]

	let blacklists = ["https://mail.google.com/*", "*://mail.google.com/*", "@https://mail.google.com/mail/*", "*://talkgadget.google.com/*", "*://typing-speed-test.aoeu.eu/*", "*://youtube.com/*"]

	" blacklists prefixed by '@' act as a whitelist

	let mapleader = "<Space>"

	" Mappings

	map <Leader>r reloadTabUncached
	map u :undo<cr>

	map gb :buffer<Space>
	map yf gy
	map myf my
	map <Leader>f createActiveTabbedHint
	map <Leader>; I
	map <A-j> zi
	map <A-k> zo

	imap <C-o> editWithVim

**Comment by borring** [comment]
This morning, I noticed that I can use <c-i> and <c-b> to _italicize_ and **bold** my words in Hangouts windows again. That's pretty much my only usecase for blacklists so far.

cvim version hasn't changed, so it must be the Chrome version. I'm on Chrome 45 now.

**Comment by fjahr** [comment]
+1 Just tried out cVim from the Chrome Store and blacklisting as well as "disabling for domain" are not working for me on Chrome 56.

**Comment by brainstorm** [comment]
Indeed, it is not working me for either on Chrome 75 with blacklist:

`let blacklists = ["*://mail.google.com/*"]`

(the only thing I have in my cVimrc so far...)

**Comment by brainstorm** [comment]
Correction, everything seems to work fine after restarting the browser!... weirdly, GMail changed the "delete email" shortcut from `d` to `#`, that confused me.

Good work @1995eaton, love you extension! :)
